python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -105,7 +105,8 @@ def __copy__(self, memo=None):
             else:
                 this_dict[elem] = None
         #obj.__setstate__(this_dict)
-        obj.getParent(self._getParent())
+        if self._getParent() is not None:
+            obj._setParent(self._getParent())
```

Comment:
OK, cool, I was just wondering if this was needed due to some bug or not that I was unaware of :D